client/service/src/config.rs

```diff
@@ -171,6 +171,14 @@ pub enum DatabaseConfig {
 	/// A custom implementation of an already-open database.
 	Custom(Arc<dyn KeyValueDB>),
 }
+/// Configuration of the database of the client.
+#[derive(Clone, Default)]
+pub struct OffchainWorkerConfig {
+	/// If this is allowed.
+	pub enabled: bool,
+	/// allow writes from the runtime to the offchain worker database
+	pub indexing_enabled: bool,
```

Comment:
@wpank @shawntabrizi  we're going to want validators to run with this flag. Are there documentation updates corresponding?